src/components/Sensors/SensorAccordion.tsx

```diff
@@ -143,14 +143,17 @@ function SensorAccordion({
   const parsedInfo = markdownConverter.makeHtml(additionalInfo);
 
   const iconSrc = icon;
+  const url = window.location.pathname
+  const sensorId = url.substring(url.lastIndexOf('/') + 1);
```

Comment:
Does the parent component have access to this ID to pass it down? Feels fragile if we can pass it down through props

Reply:
hmm SensorTaxonomy is the parent but doesn't have the Id, neither does SensorView.... still investigating

Reply:
if nothing else, it should be available as a route param, which can be passed down or re-accessed

Reply:
ok fixed, passed down sensorId from Sensor down to SensorAccordian (was a few components deep)